.travis.yml

```diff
@@ -3,8 +3,8 @@ dist: trusty
 sudo: required
 # xcode8 has jdk8
 osx_image: xcode8
-# Not technically required but suppresses 'Ruby' in Job status message.
-language: java
+# Since Bazel 0.25, PY3 is the default Python version. Make sure Travis has it.
```

Comment:
Possibly a stupid question, but if j2cl only can build on bazel 24, does this matter? 

https://github.com/google/j2cl/blob/master/.bazelversion

Reply:
You can already build j2cl with the last bazel version if you use the flag --incompatible_disable_deprecated_attr_params=false This is due to the dependency to protocol buffer.
They fix the problem https://github.com/protocolbuffers/protobuf/pull/6301 on their side and when we upgrade the version of our dependency, we may change the bazel version to the latest

Reply:
Right, not objecting necessarily to this, just pointing out that this line doesn't seem to make sense, as a few lines later we reference the contents of .bazelversion to download 24.0.

Using .bazelversion is consistently working for us in circleci as well.

Reply:
Hopefully, #67 is applied soon which will update to a later version of Bazel

Reply:
With https://github.com/google/j2cl/commit/7114599771f8a3a8fe146e3704d425be1a9ee6a4 you don't the specific blaze flag

Reply:
:+1: